pandas/tseries/tests/test_timeseries.py

```diff
@@ -1143,6 +1143,18 @@ def test_to_datetime_tz_pytz(self):
                                  dtype='datetime64[ns, UTC]', freq=None)
         tm.assert_index_equal(result, expected)
 
+    def test_to_datetime_utc_is_true(self):
+        # See gh-11934
+        start = pd.Timestamp('2014-01-01', tz='utc')
+        end = pd.Timestamp('2014-01-03', tz='utc')
+        date_range = pd.bdate_range(start, end)
+        df = pd.DataFrame({'x': 1}, index=date_range)
+        date_index = pd.to_datetime(df.index, utc=True)
+
+        expected = 'datetime64[ns, UTC]'
+        out_dtype = str(date_index.dtype)
```

Comment:
construct an actual DatetimeIndex as the expect d result and use assert_index_equal

you don't need to construct the DataFrame here as you are only using the index